group_vars/gxconfig.yml

```diff
@@ -873,6 +874,46 @@ base_app_main: &BASE_APP_MAIN
   # <config_dir>.
   #email_domain_allowlist_file: null
 
+  # E-mail ban file is used to specify email addresses that have been
+  # banned. If a user attempts to register a new account using an email
+  # address listed in this file, registration will be denied. This file
+  # does not affect user sign-in. Email addresses are matched against a
+  # canonical address representation based on rules defined in
+  # <canonical_email_rules>. The file should include one email address
+  # per line. Lines starting with the "#" character are ignored.
+  # Example value "banned_emails.conf"
+  # The value of this option will be resolved with respect to
+  # <config_dir>.
+  #email_ban_file: null
```

Comment:
@mira-miracoli @sanjaysrikakulam just to make you aware of this feature

Reply:
Is there a reason why `sign-in` was allowed and only `registration` is banned with the banned email addresses?

Reply:
As I understand it, the ban implies that the account will eventually be deleted. Still, when you delete the account, the email is now redacted from the database and then "free" again to register, so this prevents re-registering the same email that was banned.

Reply:
I guess if you add the email to the ban list, even if it prevents signing in, you still would want to delete or purge the offending user, so it does not save you a step, it just changes the order :thinking: 

Maybe you can have a script that does both things (edit the ban file and delete the user using the API) :+1: 

Here is the relevant PR for more info https://github.com/galaxyproject/galaxy/pull/19413